cypress/integration/pages/index.js

```diff
@@ -3,7 +3,50 @@ import envConfig from '../../support/config/envs';
 import appConfig from '../../../src/app/lib/config/services';
 import describeForEuOnly from '../../support/describeForEuOnly';
 
-const runCommonTests = ({ service, pageType }) => {
+export const runCommonCanonicalTests = ({ service, pageType }) => {
```

Comment:
I'd suggest putting this in a file `cypress/integration/pages/canonical.js` to make it more easily visible at the top level.

Reply:
Wouldn't the same apply to the rest of the file?

Reply:
Not quite - `runCommonTests` is used for both canonical and amp pages.

Reply:
Ok, I agree we should make it clearer, I've written that up as part 2 of https://github.com/bbc/simorgh/issues/3340 which I'll do right after this PR